src/Umbraco.Headless.Client.Net.Web/DependencyInjection/AppBuilderExtensions.cs

```diff
@@ -62,5 +63,22 @@ public static class AppBuilderExtensions
                 routes.Routes.Add(new UmbracoRouter(actionInvokerFactory, umbracoControllers));
             });
         }
+
+        public static void UseRobotsTxt(this IApplicationBuilder app)
```

Comment:
I don't think this should be part of the web library, since the web library is shipped as a nuget package. It's only for our sample (the MVC one since it's deployed to Azure), maybe we shoud just remove it entirely and then configure the `X-Robots-Tag` header for the deployed sample instead

Reply:
I think that is a good approach as well, It is not strictly necessary to get started. I will delete it from `**.Web` and the samples that use it.